### PR TITLE
Add bot for helping with standard names

### LINKED ISSUE #88: Bot to automate countdown process for standard name proposals
As suggested with some subsequent approval by others in https://github.com/cf-convention/discuss/issues/80#issuecomment-741289496, it would be beneficial to have a tool to automate the management (time-tracking, repeated announcement, pausing etc.) of time periods initiated by lack of further comment on a given issue to propose a new standard name. Specifically this was suggested about the "cooling-off" period proposed in #80 but it could be generalised to assist with any aspect relating to a triggered countdown or events dependent on the lack or otherwise of new comments.

I am yet to investigate possibilities for implementation, but off the top of my head I think there are various options including:
* writing scripts to directly query the GitHub API;
* using GitHub Actions (including re-using pre-build Actions and adapting them to our needs via a specialised workflow);
* using and customising some dedicated GitHub Issue bot library (I think it is safe to assume there are many in existence).

### PR DESCRIPTION
**Pull request by erget**
Closes #88

### CODE REVIEW

**Comment by feggleton** [comment]
I have done some work to add a bit of automation into the standard name process using GitHub Actions. I have currently added the following actions:
- When a new issue is added with the label 'standard name', a comment will be added to explain a little bit about the next steps and link to the cfeditor where name requests are added. [happy for text to edited or suggestions given]
- When an issue with the label 'standard name' has not been commented on or seen activity for 30 days (date can be amended), a label will be added of 'moderator attention' and a comment will be added to encourage discussion whilst notifying standard name managers so they can also comment [again, happy for text suggestions or changes]

I have thought about some other Actions which could be added but they are a bit more complicated so would need to be considered carefully before implementing. For example, after adding the 'accept within 7 days' label it would great to have a comment added after 7 days tagging either myself or Alison so we can set the label to accepted if that's appropriate. However, this would depend on any comments being added in that 7 days and then those comments being assessed as either agreement or opposition. I'd need to think about how this might work and whether this would actually be useful.

In the original ticket, someone had also requested 'the originator can flag a request, once it appears that consensus has been reached, so that there's no need to wait for the proposal to become inactive' - I think this would work by adding the 'moderator attention' label and tagging either myself or Alison.

I have tested as many scenarios as I could think of but would appreciate any other testing or test cases others can suggest in order to get this implemented.

File: .github/workflows/stale-issue-reminder.yml [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+    steps:
+    - uses: actions/stale@v7
+      with:
+        repo-token: ${{ secrets.GITHUB_TOKEN }}
+        # Issues or Pull Requests with these labels will never be considered stale. Set to `[]` to disable
+        #exempt-issue-labels:
+           #-'accept within 7 days'
+        days-before-close: -1   # disabled
+        only-issue-labels: standard name
+        stale-issue-message: 'This issue has had no activity in the last 30 days. This is a reminder to please comment on standard name requests to assist with agreement and acceptance. Standard name moderators are also reminded to review @feggleton'
```
Comment (feggleton) [inline .github/workflows/stale-issue-reminder.yml:24]: The comment here will need to be changed to include a tag for Alison if she is ok with that. I have only added a tag for myself for now for the purposes of testing as to not attack Alison's inbox.

**Comment by sadielbartholomew** [comment]
Brilliant stuff @feggleton, thanks for getting this implemented! I will try to review this by early next week at the latest.

**Review by sadielbartholomew — COMMENTED** [review]
I've started some testing on a dummy repo (see https://github.com/sadielbartholomew/crash_test_dummy/issues/1 if interested), but whilst I wait a few days for any triggering adjusted to occur for stale issues, so I can review whether or not that is working as it should be, here are a few minor formatting comments. Everything looks good in terms of the configuration by eyeball, otherwise!

File: .github/label-actions.yml [hunk 2]
```diff
@@ -0,0 +1,8 @@
+
+
+standard name:
+    comment: |
+        Thank you for your proposal. These terms will be added to the cfeditor (http://cfeditor.ceda.ac.uk/proposals/1) shortly. Your proposal will then be reviewed and commented on by the community and Standard Names moderator.
+
+
+
```
[the remark this answers is not in the document]
Comment (feggleton) [reply .github/label-actions.yml:8]: yes that's fine - i think I started adding another one but removed it.

**Review by sadielbartholomew — APPROVED** [review]
From testing over on https://github.com/sadielbartholomew/crash_test_dummy/issues/1 and the conversation there, we've observed that everything works as it should, and my minor feedback regarding superfluous whitespace has been addressed.

So, all good (though ideally we can squash down all of the ~35 commits here to <5 self-contained ones, to keep the commit history clean).  Thanks for all of your work on this, @feggleton! When @japamment is happy, let's get this merged.

**Comment by feggleton** [comment]
Thanks for your review @sadielbartholomew I'm fairly happy with the testing but as we merge (and happy to squash), I'd like to post on the issues tab an announcement to say what will happen going forward and mention this might be a trial period so anyone can raise issues or say if they think bits aren't working well.

**Comment by feggleton** [comment]
Alison has stated she is happy to proceed here so I'd like to go ahead and merge this. Let the trial begin. Thanks

**Pull request merged**